crates/driver/src/boundary/mod.rs

```diff
@@ -42,12 +43,25 @@ pub use {
     },
 };
 
-use crate::infra::blockchain::Ethereum;
-
 /// Returns a Web3 instance with a trait object transport needed by various
 /// boundary components.
 fn web3(eth: &Ethereum) -> Web3 {
     // Ugly way to get access to one of these... However, this way we don't
     // leak this into our domain logic.
     eth.contracts().settlement().raw_instance().web3()
 }
+
+/// Builds a web3 client that bufferes requests and sends them in a
+/// batch call.
+pub fn buffered_web3_client(ethrpc: &Url) -> Web3 {
```

Comment:
nit: Aren't we reading these arguments from the config usually (with sensible defaults). If we hardcode defaults here, I worry we might forget updating these values in all places when we want to make a change.

Reply:
Yes usually we read these value from some configuration. However, these values have not been changed in forever so I though it a reasonable intermediate solution to just hard code the battle tested values here.
I think Nick's suggestion of moving the code into a crate and depend on that is a reasonable next step and I would prefer nailing down the configurations options then.